.travis.yml

```diff
@@ -21,6 +21,15 @@ jobs:
         - CONTAINER_RUNTIME=docker
         - NETWORK_PROVIDER=weave
         - FOOTLOOSE_IMAGE=quay.io/footloose/ubuntu18.04
+    - stage: e2e
+      name: "e2e: focal docker with weave"
+      script: ./e2e/travis.sh
+      rvm: 2.7.1
+      dist: focal
```

Comment:
This needs to be bionic (it's the host OS where we execute tests within docker).

Reply:
Thanks :) I've just pushed this change